src/ProgressMeter.jl

```diff
@@ -580,11 +580,21 @@ function printvalues!(p::AbstractProgress, showvalues; color = :normal, truncate
     p.numprintedvalues = 0
 
     for (name, value) in showvalues
-        msg = "\n  " * lpad(string(name) * ": ", maxwidth+2+1) * string(value)
+        string_value = "\e[0m" * if value isa Union{String, SubString{String}}
+            value
+        else
+            repr("text/plain", value; context=IOContext(PipeBuffer(), :color => true))
+        end
+        if countlines(IOBuffer(string_value)) > 1
+            # Multiline objects should go on their own line so their
+            # alignment doesn't get messed up
+            string_value = "\n" * string_value
+        end
+        msg = "\n  " * lpad(string(name) * ": ", maxwidth+2+1) * string_value
         max_len = (displaysize(p.output)::Tuple{Int,Int})[2]
         # I don't understand why the minus 1 is necessary here, but empircally
         # it is needed.
```

Comment:
Does this "empirically it is needed" change at all with this modified calculation? Is there a test for this that verifies it is/is not needed?

Reply:
That's an old comment from before my PR. I did try some tests to change that value, and it causes the number of printed lines to shrink or expand each iteration.

Reply:
Yeah I assumed it was old, I was just wondering if it was no longer relevant since it’s now a different method for counting lines